README.rst

```diff
@@ -224,6 +224,66 @@ is no natural place for the annotation to live. Here is one possibility:
     class Leaf(Node): ...
     class Branch(Node): ...
 
+Just use Union
```

Comment:
Should Union be monospaced here as ``Union``?

Reply:
Yep